modules/csg/editor/csg_gizmos.h

```diff
@@ -37,8 +37,11 @@
 
 #include "editor/plugins/editor_plugin.h"
 #include "editor/plugins/node_3d_editor_gizmos.h"
+#include "scene/gui/option_button.h"
```

Comment:
Why OptionButton? You could include `control.h`.

Reply:
That plugin/gizmo/editor(control) is how many 3D node plugins are done. Some older I have found even comments that they should be changed to use a Control instead of having the plugin or gizmo full of gui related stuff so not sure what is the general accepted take on this, I just copied mostly from existing plugins.

Reply:
> Some older I have found even comments that they should be changed to use a Control instead of having the plugin or gizmo full of gui related stuff

Which one?

Reply:
> // TODO: Split the EditorPlugin instance from the UI instance and connect this properly.

E.g. found on the Path3d editor plugin.

Reply:
idk, the Path3D editor is an actual editor, not a single button with 2 methods. I think it's not worth it to make it into a separate class, especially Node-based one, as it doesn't really use any Node methods. The THEME_CHANGED notification can be replaced by signal connection like in path editor. In theory it's nice to separate "editor" functions into a new class, but if it's still the same file, then it doesn't really matter. Though I guess if you *really* want to make a separate class, make it inherit OptionButton, so the editor is the button that you add to the toolbar.